app-web/src/components/TopicPreview/TopicContent.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import styled from '@emotion/styled';
+import { faChevronDown } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+const Title = styled.h3`
+  font-size: 1rem;
+  margin: 0;
+  font-weight: 300;
+  color: #121112;
+`;
+
+const Inline = styled.div`
+  display: block;
+  padding: 0.4rem 0.8rem;
+  display: flex;
+  justify-content: space-between; //for arrowIcon
+  align-items: center;
+  background-color: #fff;
+  z-index: 2;
+  position: center;
+`;
+const ArrowIcon = styled.span`
+  width: 1.2rem;
+  height: 1.2rem;
+  transition: transform 0.3s ease-in-out;
+  background-size: contain;
+  background-repeat: no-repeat;
+  opacity: 0.8;
+  transform-origin: 30% 50%;
+  transform: ${props => (props.opened ? 'rotate(0deg)' : 'rotate(180deg)')};
+`;
+const Inner = styled.div`
+  max-height: 200rem;
+  background: #dcdcdc;
+  border-radius: 10px;
+`;
+const Content = styled.div`
+  opacity: 1;
+  margin-left: 30px;
+`;
+
+const TopicContent = ({ topics, ...rest }) => {
+  const [opened, setOpened] = useState(false);
+  return (
+    <div>
+      <Inline onClick={() => setOpened(!opened)}>
+        <Title>{topics.name}</Title>
+        <ArrowIcon opened={opened}>
+          <FontAwesomeIcon icon={faChevronDown} />
+        </ArrowIcon>
+      </Inline>
+      {opened && (
+        <Inner>
+          {topics.childrenDevhubSiphon.map(item => (
+            <Content>
+              <a href={item.resource.path}>{item.fields.title}</a>
```

Comment:
again to avoid coupling this to a data structure, it is better if the list of links is more simplified. Perhaps the list of topic connections is represented as `[{to: '...', text: '...'}]`